lib/presentation/map_tokyo/map_tokyo_page.dart

```diff
@@ -0,0 +1,140 @@
+                .map((mapTokyoWardButtonInformation) => FlatButton(
+                      color: mapTokyoWardButtonInformation.disFlag
+                          ? Theme.of(context).primaryColor
+                          : Colors.grey,
+                      child: Text(
+                        mapTokyoWardButtonInformation.tokyoWardName,
+                        style: TextStyle(
+                          fontWeight: FontWeight.bold,
+                          fontSize: 9,
+                          color: Colors.white,
+                        ),
+                      ),
+
+                      //TODO 23区押下後の達成リストに遷移
+                      onPressed: () {},
+                    ))
+                .toList();
+            mapInfo.insert(0, Image.asset("images/tokyo/whitetokyo.png"));
+            return Stack(
+              children: [
+                Center(
+                  child: Column(children: [
+                    SizedBox(
+                      height: 8.0,
+                    ),
+                    Stack(
+                      children: [
+                        Row(
+                          mainAxisAlignment: MainAxisAlignment.center,
+                          children: [
+                            Text(
+                              '達成数：',
+                              style: TextStyle(
+                                  fontWeight: FontWeight.bold, fontSize: 16),
+                            ),
+                            Text(
+                              '${model.completeNumberOfTokyoWard}',
+                              style: TextStyle(
+                                  fontWeight: FontWeight.bold,
+                                  fontSize: 28,
+                                  color: Theme.of(context).accentColor),
+                            ),
+                            SizedBox(
+                              width: 16.0,
+                            ),
+                            Text(
+                              '残り：',
+                              style: TextStyle(
+                                  fontWeight: FontWeight.bold, fontSize: 16),
+                            ),
+                            Text(
+                              '${model.remainNumberOfTokyoWard}',
+                              style: TextStyle(
+                                  fontWeight: FontWeight.bold,
+                                  fontSize: 28,
+                                  color: Theme.of(context).accentColor),
+                            ),
+                          ],
+                        ),
+                      ],
+                    ),
+                    Expanded(
+                      child: Center(
+                        child: Stack(
+                          children: mapInfo,
+                        ),
+                      ),
+                    ),
+                    Padding(
+                      padding: const EdgeInsets.symmetric(horizontal: 24.0),
```

Comment:
```suggestion
                      padding: const EdgeInsets.only(top: 0, bottom: 32.0, right: 2.0, left: 2.0),
```

各区のボタンが小さい印象なので、mapPageと同じサイズ感でいいかと思います！

前
![unnamed (5)](https://user-images.githubusercontent.com/58384546/112945829-be989480-916f-11eb-8a52-c5e3ae966dea.jpeg)

後
![unnamed (5-3)](https://user-images.githubusercontent.com/58384546/112945876-ceb07400-916f-11eb-8f9c-9a577eacdb42.jpeg)